cmd/opampsupervisor/testdata/supervisor/supervisor_basic.yaml

```diff
@@ -0,0 +1,14 @@
+server:
+  endpoint: ws://{{.url}}/v1/opamp
+  tls:
+    insecure: true
+
+capabilities:
+  reports_effective_config: true
+  reports_own_metrics: true
+  reports_health: true
+  accepts_remote_config: true
+  reports_remote_config: true
+
+agent:
+  executable: ../../bin/otelcontribcol_{{.goos}}_{{.goarch}}{{.extension}}
```

Comment:
Do the tests require the presence of this executable? How do we ensure the executable exists when we only do `go test` to run the tests locally? Do we expect there to be a `make` target that helps running these tests on a developer machine?

Reply:
Good call. I added a `e2e-test` Make target in the directory that handles building the Collector.